assets/tutorials/multiple_feasible_solutions.md

```diff
@@ -0,0 +1,305 @@
+```@meta
```

Comment:
A few things:

 * You ran this with `Literate(documenter = true)` right? The JuMP website doesn't run Documenter, so you need to drop the `documenter` argument.  That should get rid of this URL, and it'll paste the output of the `@example` blocks into the Markdown.
 * This file should be in `/_posts/` and be dated appropriately
 * You'll have to manually copy an appropriate header. Something like
```
---
layout: post
title: "Finding multiple feasible solutions"
date: 2021-11-01
categories: [tutorials]
---
```